projects/plugins/jetpack-mu-wpcom/phpunit.xml.dist

```diff
@@ -3,6 +3,9 @@
 		<testsuite name="main">
 			<directory prefix="test" suffix=".php">tests/php</directory>
 		</testsuite>
+		<testsuite name="coming-soon">
+			<directory suffix="-test.php">./src/features/coming-soon/test/</directory>
```

Comment:
Any particular reason not to put the tests into `tests/php/` while we're moving stuff around?

Reply:
Moved the test: a907b72ef029d118e13b669ba4ccd75828956eb6